src/cryptsetup/cryptsetup.c

```diff
@@ -1988,7 +1988,8 @@ static int run(int argc, char *argv[]) {
                 if (until == USEC_INFINITY)
                         until = 0;
 
-                arg_key_size = (arg_key_size > 0 ? arg_key_size : (256 / 8));
+                if (arg_key_size == 0)
+                        arg_key_size = 256U / 8U;
```

Comment:
Bit off topic but we should be using larger keys these days supposedly, see https://github.com/systemd/systemd/pull/24944#discussion_r994390109